src/PrestaShopBundle/Form/Admin/Extension/MultistoreConfigurationTypeExtension.php

```diff
@@ -76,4 +83,27 @@ public static function getExtendedTypes(): iterable
     {
         return [MultistoreConfigurationType::class];
     }
+
+    /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        parent::configureOptions($resolver);
+
+        if (!$this->multistoreCheckboxEnabler->shouldAddMultistoreElements()) {
+            return;
+        }
+
+        $resolver->setNormalizer('attr', function (Options $options, array $value): array {
+            $classes = 'js-multishop-form-row';
+            if (!empty($value['class'])) {
+                $classes .= ' ' . $value['class'];
+            }
+
+            $value['class'] = $classes;
```

Comment:
```suggestion
            $value['class'] = 'js-multishop-form-row';
            if (!empty($value['class'])) {
                $value['class'] .= ' ' . $value['class'];
            }
```

Reply:
Hmm no @Progi1984  your suggestion is removing the initial value on first line so you can't use it after But it could be simplified indeed

Reply:
```suggestion
            $value['class'] = trim('js-multishop-form-row ' . ($value['class'] ?? ''));
```